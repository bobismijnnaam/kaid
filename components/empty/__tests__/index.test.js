import React from 'react';
import { shallow, render, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json';

import Empty from '..';

configure({ adapter: new Adapter() });

describe('Empty view', function() {
  it('renders correctly', () => {
    const wrapper = shallow(<Empty text="empty"/>);
    expect(toJson(wrapper)).toMatchSnapshot();
  });


  it('should have class "kai-empty"', function() {
    expect(shallow(<Empty />).is('.kai-empty')).toBe(true);
  });

  it('should not able to render its children', function() {
    expect(render(<Empty>empty</Empty>).text()).toEqual('');
  });
});
