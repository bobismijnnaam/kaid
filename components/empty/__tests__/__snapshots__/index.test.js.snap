// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Empty view renders correctly 1`] = `
<div
  className="kai-empty"
>
  <p
    className="kai-empty-text"
    data-l10n-id="empty"
  />
</div>
`;
